discovery-provider/alembic/versions/c8d2be7dcccc_repair_poorly_sorted_tracks.py

```diff
@@ -12,20 +12,187 @@
 
 """
 import logging
-from alembic import op
+from typing import Any
 from datetime import datetime
-from sqlalchemy import Integer
+from jsonschema import ValidationError
+from sqlalchemy.dialects import postgresql
 from sqlalchemy.ext.compiler import compiles
+from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.sql.functions import GenericFunction
-from sqlalchemy.orm import sessionmaker
-from src.models import Track
+from sqlalchemy.orm import sessionmaker, validates
+from sqlalchemy.sql import null
+from sqlalchemy import (
+    Column,
+    Integer,
+    String,
+    Boolean,
+    DateTime,
+    ForeignKey,
+    Text,
+    PrimaryKeyConstraint,
+)
+from alembic import op
+from src.model_validator import ModelValidator
 
 # revision identifiers, used by Alembic.
 revision = "c8d2be7dcccc"
 down_revision = "d9992d2d598c"
 branch_labels = None
 depends_on = None
 
+Base: Any = declarative_base()
```

Comment:
Maybe add a quick comment explaining to the future reader why we had to steal all this code

Reply:
How's line 113?

Reply:
ah i missed that. 👍